src/resolvers/admin.ts

```diff
@@ -23,4 +24,20 @@ export class AdminResolver {
     await killSwitch.save()
     return { isKilled }
   }
+
+  @Mutation(() => GeneratedSignature)
+  async generateSignature(@Arg('bytes', () => String) bytes: string) {
+    if (!config.appPrivateKey) {
+      throw new Error('No PRIVATE_KEY provided to generate signature')
+    }
```

Comment:
`loadEnvVar` will throw if this variable is missing so I don't think we need this check here